plugin/quicklink.vim

```diff
@@ -95,4 +95,30 @@ function! OpenLinkOnCurrentLine()
   call system('open ' . expand('<cWORD>'))
 endfunction
 
+function! s:OpenLink()
+  "get the initial position of the curosr
+  let initial_pos = getpos('.')
+  "empty the @a register in the case cursor is not well palced and that old
+  "patterns would still trigger.
+  let @a = ""
+  normal "ayi]
+  let escaped_link_name = escape(getreg('a'), '&')
+  let pattern = '\v^\['.escaped_link_name.'\]: (%(ftp[s]?|http[s]?):\/\/\S+)>'
```

Comment:
Can you rename this variable to be a bit more explicit? `link_target_pattern` or similar?